mailpoet/mailpoet
# Replace some of the usages of the raw filter in the Twig templates

## Description

This PR replaces some of the usages of the raw filter in the Twig templates with more secure alternatives. See the commit messages for more details on each of the approaches.

It seems to me that there is not much we can do about the other uses of `| raw` that are still left in the code base. The variables that are being passed to the `raw` filter contain complex HTML that I'm not sure if we could escape or filter somehow. But please let me know if I missed something and you have a suggestion.

## Code review notes

I'm not an expert on Twig templates, so I'm not sure if the approach that I took here is the best or if there is something else that we should consider. I did search a bit, and I could not find anything better.

## QA notes

This PR should have no user facing changes. But it makes some modifications that could break things in the forms, the pages to import and export subscribers, the upgrade, and the manage where subscribers can manage their subscriptions. If possible, please check those pages are still working as expected.

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5238]

## After-merge notes

_N/A_


[MAILPOET-5238]: https://mailpoet.atlassian.net/browse/MAILPOET-5238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
